internal/framework/events/first_eventbatch_preparer_test.go

```diff
@@ -58,7 +58,7 @@ var _ = Describe("FirstEventBatchPreparer", func() {
 			batch, err := preparer.Prepare(context.Background())
 
 			Expect(batch).Should(BeEmpty())
-			Expect(err).Should(BeNil())
+			Expect(err).ShouldNot(HaveOccurred())
```

Comment:
```suggestion
			Expect(err).ToNot(HaveOccurred())
```
For consistency with the rest of our tests.